src/bgw/job_stat.h

```diff
@@ -14,14 +14,6 @@ typedef struct BgwJobStat
 	FormData_bgw_job_stat fd;
 } BgwJobStat;
 
-/* Positive result numbers reserved for success */
```

Comment:
Please revert the change here. Not needed.

Reply:
Removed this change.